### PR TITLE
Added claymore as a weapon

### PR DESCRIPTION
**Pull request by TaralGit**
Added the claymore as my last contribution for melee weapons. Also, my last adjustments for the combat knife, laser gun and machete sprites.

Screenshot 2021-06-01 043026claymore
Claymore sprite and inhands

Update:

Screenshot 2021-06-04 193734claymore
updated claymore sprite

Update2:
Screenshot 2021-06-08 034605lastsprites
All the sprites featured

:cl:
- add: claymore as a melee weapon
- fix: fixed the combat knife, laser gun and machete sprites

### CODE REVIEW

File: Resources/Textures/Objects/Weapons/Melee/claymore.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Taken from tgstation and modified by Taral at https://github.com/tgstation/tgstation/pull/31615/commits/1ffa93a49df1e728a4f55d6b867879d29a8bec1a",
```
Comment (mirrorcult) [inline Resources/Textures/Objects/Weapons/Melee/claymore.rsi/meta.json:4]: link didn't work for me, should it be https://github.com/tgstation/tgstation/pull/31615/commits/8795484578b33a9919b7801667287c0f28afe642?

**Review by mirrorcult — APPROVED** [review]

File: Resources/Textures/Objects/Weapons/Melee/claymore.rsi/meta.json (hunk 1 above)
Comment (mirrorcult) [reply Resources/Textures/Objects/Weapons/Melee/claymore.rsi/meta.json:4]: or https://github.com/tgstation/tgstation/pull/31400/commits/6046fb493bf487a679a235acfebfe3674cf2db8a?

Comment (mirrorcult) [reply Resources/Textures/Objects/Weapons/Melee/claymore.rsi/meta.json:4]: everything else is good btw

**Comment by SweptWasTaken** [comment]
Why do you have a commit called machete? Also i'm going to be honest I don't like how the claymore looks. Looks like some kind of light blade instead of metal

File: Resources/Textures/Objects/Weapons/Melee/claymore.rsi/meta.json (hunk 1 above)
Comment (TaralGit) [reply Resources/Textures/Objects/Weapons/Melee/claymore.rsi/meta.json]: ok. I fixed the commit from the meta.json

**Comment by TaralGit** [comment]
> Why do you have a commit called machete? Also i'm going to be honest I don't like how the claymore looks. Looks like some kind of light blade instead of metal

that's for the machete sprite. I will fix the claymore tomorrow by giving the blade a more metalic look.

**Comment by TaralGit** [comment]
ok. it's all done

**Comment by SweptWasTaken** [comment]
> 
> 
> ok. it's all done

Can you show your changes?

**Comment by TaralGit** [comment]
> > ok. it's all done
> 
> Can you show your changes?

Ok. I updated the OP with another screenshot with all the sprites that we're worked on.

**Comment by SweptWasTaken** [comment]
The claymore looks great now.

**Review by SweptWasTaken — APPROVED** [review]
?

You removed these laser gun and machete sprites but added the combat knife sprite? If you're going other stuff like this in a PR you need to write it in your description.

**Comment by TaralGit** [comment]
> ?
> 
> You removed these laser gun and machete sprites but added the combat knife sprite? If you're going other stuff like this in a PR you need to write it in your description.

Yeah, I basically fixed the sprites for the combat knife, machete, laser gun and added the claymore. The end result is in the last bottom screenshot and I also added all the changes in the changelog, but in future pulls I will describe my commits a little better than just "last touches" or "." .

**Pull request merged**